### PR TITLE
Fix Istiocli typo when printing warning for IST0151

### PR DESCRIPTION
**Pull request by bordenit**
change "of" to "or", see https://istio.io/latest/docs/reference/config/analysis/ist0151/. This change matches the wording of the description on that page.

**To help us figure out who should review this PR, please put an X in all the areas that this PR affects.**

- [ ] Ambient
- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [x] User Experience
- [ ] Developer Infrastructure

**Please check any characteristics that apply to this pull request.**

- [x] Does not have any [user-facing](https://github.com/istio/istio/tree/master/releasenotes#when-to-add-release-notes) changes. This may include CLI changes, API changes, behavior changes, performance improvements, etc.

### CODE REVIEW

**Comment by bordenit** [comment]
CLA has been signed, and it is a one word change, so there may not be a test to change?

**Review by howardjohn — COMMENTED** [review]
lgtm but this is generated from messages.yaml so we would need to update that

/ok-to-test

File: pkg/config/analysis/msg/messages.gen.go [hunk 1]
```diff
@@ -199,7 +199,7 @@ var (
 
 	// EnvoyFilterUsesRelativeOperation defines a diag.MessageType for message "EnvoyFilterUsesRelativeOperation".
 	// Description: This EnvoyFilter does not have a priority and has a relative patch operation set which can cause the EnvoyFilter not to be applied. Using the INSERT_FIRST or ADD option or setting the priority may help in ensuring the EnvoyFilter is applied correctly.
-	EnvoyFilterUsesRelativeOperation = diag.NewMessageType(diag.Warning, "IST0151", "This EnvoyFilter does not have a priority and has a relative patch operation set which can cause the EnvoyFilter not to be applied. Using the INSERT_FIRST of ADD option or setting the priority may help in ensuring the EnvoyFilter is applied correctly.")
+	EnvoyFilterUsesRelativeOperation = diag.NewMessageType(diag.Warning, "IST0151", "This EnvoyFilter does not have a priority and has a relative patch operation set which can cause the EnvoyFilter not to be applied. Using the INSERT_FIRST or ADD option or setting the priority may help in ensuring the EnvoyFilter is applied correctly.")
```
Comment (ymesika) [inline pkg/config/analysis/msg/messages.gen.go:202]: This is a generated file but I wonder if `..option or setting the priority..` should actually be `option for setting the priority`.
@howardjohn ?

Comment (hzxuzhonghu) [reply pkg/config/analysis/msg/messages.gen.go:202]: No, here priority is a new field of envoyfilter.

**Comment by hanxiaop** [comment]
not stale

File: pkg/config/analysis/msg/messages.gen.go (hunk 1 above)
Comment (hanxiaop) [inline pkg/config/analysis/msg/messages.gen.go:202]: Can you edit the `messages.yaml` file in the same directory? This file is generated from the yaml file.

**Pull request closed without merging**